src/views/activity/activity.view.tsx

```diff
@@ -86,36 +86,48 @@ const Activity: FC = () => {
           });
         })
         .finally(() => {
-          setAreBridgesDisabled(false);
+          if (isAsyncTaskDataAvailable<Bridge[], undefined, true>(apiBridges)) {
+            getPendingBridges(apiBridges.data)
+              .then((data) => {
+                callIfMounted(() => {
+                  setPendingBridges({ status: "successful", data });
+                });
+              })
+              .catch((error) => {
+                callIfMounted(() => {
+                  notifyError(error);
+                });
+              })
+              .finally(() => setAreBridgesDisabled(false));
+          }
         });
     }
   };
 
   const processFetchBridgesSuccess = useCallback(
     (bridges: Bridge[]) => {
-      void getPendingBridges(bridges).then((pendingBridges) => {
-        callIfMounted(() => {
-          const filteredApiBridges = bridges.filter((bridge) => {
-            const isBridgePresentInPendingBridges =
-              (bridge.status === "pending" || bridge.status === "on-hold") &&
-              pendingBridges.find(
-                (pendingBridge) => pendingBridge.depositTxHash === bridge.depositTxHash
-              ) !== undefined;
-            return isBridgePresentInPendingBridges ? false : true;
+      setLastLoadedItem(bridges.length);
```

Comment:
What is the difference between `total` and `lastLoadedItem`? Shouldn't they be the same?

Reply:
* `total`: How many items there are in the collection
* `lastLoadedItem`: How many items have been loaded so far